src/components/ScoreboardCard.tsx

```diff
@@ -35,7 +35,20 @@ export default function ScoreboardCard(props: ScoreboardCardProps) {
                 src={`${play.sport}-Logo.png`}
                 alt={`${play.sport} logo`}
               />
-              <Avatar src={playerImageSrc(play)} alt={play.player_name} />
+              <Avatar src={playerImageSrc(play)} alt={play.player_name}>
+                {
+                  // fallback image
+                  <img
+                    style={{
+                      width: "100%",
+                      height: "100%",
+                      objectFit: "cover",
+                    }}
+                    src="https://a.espncdn.com/combiner/i?img=/i/headshots/nophoto.png&h=110&w=110&scale=crop"
+                    alt={play.player_name}
+                  />
+                }
```

Comment:
This is weird. Why the curly braces? Why not pass it as a named prop like `fallbackImg`?

Reply:
Was following this: https://stackoverflow.com/questions/63276812/how-to-change-with-fallback-image-with-material-ui. i.e. Avatar falls back to its children, you can't pass a fallback as a prop.